### DISCUSSION TITLE
fa/manager/installation-and-setup/Quick-Installation-on-OVH-Servers/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4543] [Content Help کمک به اصلاح محتوا]
# fa/manager/installation-and-setup/Quick-Installation-on-OVH-Servers/

The proxy manager platform.

https://hiddify.com/fa/manager/installation-and-setup/Quick-Installation-on-OVH-Servers/

**Comment by kayarsalan** [comment]
سلام متاسفانه بعد از کامل شدن نصب روی و مراجعه به آی پی با این ارور مواجه میشم:
dial tcp4 84.17.42.48:0->91.134.87.73:80: connect: connection refused

**Comment by AlexMaksyuk** [comment]
Спасибо за хорошую работу!

**Comment by fathimkmkk-wq** [comment]
#cloud-config
package_upgrade: true
packages:
  - apt-transport-https
  - ca-certificates
  - curl
  - wget
  - gnupg-agent
  - software-properties-common
  - git
  - jq

runcmd:
  - cd /opt
  - bash -c "export CREATE_EASYSETUP_LINK='true';curl i.hiddify.com/release|bash -s -- --no-gui"

final_message: "The system is finally up, after $UPTIME seconds"
output: { all: "| tee -a /root/cloud-init-output.log" }

# you can see the generated link from the website by using 
# https://yourip.sslip.io/hiddify in one hour, after that,
# it will be disappear.
